app/javascript/logs/components/data_renderers/TextLog.vue

```diff
@@ -1,7 +1,7 @@
 <template lang="pug">
-.flex.justify-center
-  .container
-    table.text-log-table
+div
+  .flex.justify-center
+    table.text-log-table.max-w-4xl
```

Comment:
This is a bit narrower than what we had before. I want to try it out. Narrow can be nice, I think, because it means less horizontal eye movement.